template/{{ common_path }}/src/main/resources/assets/{{ modid }}/lang/en_us.json.jinja

```diff
@@ -6,7 +6,7 @@
   "hexcasting.spell.book.{{ modid }}:signum": "Dum. Actn.",
   "{{ modid }}.entry.dummy_spells": "Dummy Spells",
   "{{ modid }}.entry.dummy_actions": "Dummy Actions",
-  "{{ modid }}.page.dummy_spells.congrats": "Accepts a player entity, tells them they are doing a good job.",
+  "{{ modid }}.page.dummy_spells.congrats": "Accepts a player entity, tells them they are doing a good job and makes them look up.",
```

Comment:
Maybe we should actually take the opportunity to fix this text - doesn't OpCongrats operate directly on the caster?

Reply:
Fair point, it really does. That's my oversight in the implementation of the spell, though, because it accepts a player entity (to show an example of a mishap when an incorrect entity is passed) and was supposed to act on that entity. I will be fixing that, thank you for pointing it out.